classic/02 Installation/02 Installing the Amazon Connect Salesforce Lambda Package.md

```diff
@@ -382,6 +382,12 @@ retrieve secrets.
         create one for you. This parameter in mandatory certain
         features, please see above.
 
+    k.  *SalesforceHost:* The full domain for your salesforce org. For
+        example
+        `https://mydevorg-dev-ed.my.salesforce.com`.
+        Please make sure that the host starts with `https`, and that the url
+        ends with `.my.salesforce.com`.
+
```

Comment:
Can you add a little guidance how how to get these? For example it can be found from setup -> my domain.

Reply:
done